spec/requests/api/stateless/subscriptions_spec.rb

```diff
@@ -6,9 +6,29 @@
 
   let!(:member) { create(:member, email: 'test@example.com') }
   let!(:customer) { create(:payment_braintree_customer, member: member) }
-  let!(:subscription) { create(:payment_braintree_subscription, customer: customer) }
-  let!(:transaction_a) { create(:payment_braintree_transaction, subscription: subscription ) }
-  let!(:transaction_b) { create(:payment_braintree_transaction, subscription: subscription ) }
+  let!(:payment_method) { create(:braintree_payment_method,
+    customer: customer,
+    instrument_type: 'credit card',
+    token: '2ewruo4i5o3',
+    last_4: '2454',
+    email: customer.email,
+    card_type: 'Mastercard')}
+
+  let!(:subscription) { create(:payment_braintree_subscription,
+    id: 1234,
+    customer: customer,
+    payment_method: payment_method,
+    amount: 4,
+    billing_day_of_month: 22,
+    created_at: Time.now) }
+  let!(:transaction_a) { create(:payment_braintree_transaction,
+    subscription: subscription,
+    status: 'failure',
+    amount: 100) }
+  let!(:transaction_b) { create(:payment_braintree_transaction,
```

Comment:
I think a single transaction is enough. That a subscription has many transactions should be tested at the model layer. Ditching the second transaction will also help keep the important details of this spec apparent to the reader.